src/classes/VOL_SharedCode.cls

```diff
@@ -687,7 +687,7 @@ global with sharing class VOL_SharedCode {
                 //IsManagedCode = (token != null);
 
     			Map<String, Schema.SObjectField> M = Schema.SObjectType.Contact.fields.getMap();
-    			Schema.SObjectField fld = M.get('gw_volunteers__volunteer_status__c');
+    			Schema.SObjectField fld = M.get(StrTokenNSPrefix('volunteer_status__c'));
```

Comment:
I think you just broke this!  by dynamically adding the namespace, or leaving it out when not needed, then won't the routine end up always saying IsManagedCode = true!